Queueomatic/Server/Endpoints/ResetPassword/ResetPasswordRequest.cs

```diff
@@ -0,0 +1,5 @@
+﻿using Queueomatic.Shared.DTOs;
+
+namespace Queueomatic.Server.Endpoints.ResetPassword;
+
+public record ResetPasswordRequest(ResetPasswordDto request);
```

Comment:
* This request needs to be validated for preventing storing invalid passwords.
* `request` should start with a capital R, for following the name convention.